bundles/org.openhab.binding.dali/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dali"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="daliserver">
+		<label>Daliserver</label>
+		<description>A running daliserver.</description>
+
+		<representation-property>UDN</representation-property>
+
+		<config-description-ref uri="thing-type:dali:daliserver"/>
+	</bridge-type>
+
+	<!-- Single Device Type -->
+	<thing-type id="device">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="daliserver"/>
+		</supported-bridge-type-refs>
+		<label>DALI Device</label>
+		<description>Controls a single device/ballast</description>
+		<channels>
+			<channel id="dimAtFadeRate" typeId="dimmer"/>
+			<channel id="dimImmediately" typeId="dimmer"/>
+		</channels>
+
+		<config-description>
+			<parameter name="targetId" type="integer" required="true" min="0" max="63">
+				<label>Device ID</label>
+				<description>Address of device in the DALI bus</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<!-- Group Device Type -->
+	<thing-type id="group">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="daliserver"/>
+		</supported-bridge-type-refs>
+		<label>DALI Group</label>
+		<description>Controls a group of devices/ballasts</description>
+		<channels>
+			<channel id="dimAtFadeRate" typeId="dimmer"/>
+			<channel id="dimImmediately" typeId="dimmer"/>
+		</channels>
+
+		<config-description>
+			<parameter name="targetId" type="integer" required="true" min="0" max="31">
+				<label>Group ID</label>
+				<description>Address of group in the DALI bus</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="dimmer">
```

Comment:
There are predefined system channels you could use. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types

Same for the color channel.